src/handlers/wallet/get_exchanges.rs

```diff
@@ -0,0 +1,96 @@
+use {
+    crate::handlers::wallet::exchanges::{get_supported_exchanges, Exchange},
+    crate::{
+        handlers::{SdkInfoParams, HANDLER_TASK_METRICS},
+        state::AppState,
+    },
+    axum::{
+        extract::{ConnectInfo, Query, State},
+        Json,
+    },
+    hyper::HeaderMap,
+    serde::{Deserialize, Serialize},
+    std::{net::SocketAddr, sync::Arc},
+    thiserror::Error,
+    wc::future::FutureExt,
+};
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct GetExchangesRequest {
+    pub page: u32,
```

Comment:
Use `usize` so you don't need to truncate the vec length into the u32 later

```suggestion
    pub page: usize,
```